### PR TITLE
Avoid asking for fake consent

### PR DESCRIPTION
**Pull request by dmarti**
Fake consent is worse than no consent, because of privacy labour and distraction from consent requests that should get careful attention.

***
<a href="https://pr-preview.s3.amazonaws.com/w3ctag/privacy-principles/pull/190.html" title="Last updated on Nov 2, 2022, 12:06 AM UTC (0c5256e)">Preview</a> (<a href="https://pr-preview.s3.amazonaws.com/w3ctag/privacy-principles/pull/190.html#opt-in-out" title="#opt-in-out">#opt-in-out</a>) (<a href="https://pr-preview.s3.amazonaws.com/w3ctag/privacy-principles/pull/190.html#consent-principles" title="#consent-principles">#consent-principles</a>) | <a href="https://pr-preview.s3.amazonaws.com/w3ctag/privacy-principles/190/6fffbb6...0c5256e.html" title="Last updated on Nov 2, 2022, 12:06 AM UTC (0c5256e)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -549,6 +549,14 @@
 that the person can easily switch it off. In general, providing [=consent=] should be rare,
 difficult, highly intentional, and temporary.
 
+In some situations, a [=party=] can comply with a requirement to get a [=person=] to perform
```
Comment (jyasskin) [inline index.html:552]: We should probably say that these are legal requirements. But then the rest of the advice here is hard to follow, since the site is legally required to disobey it. Maybe something like:

> There are some laws that require websites to ask their users' consent to store information even when that information doesn't affect those people's privacy. These laws are harmful. Except to comply with legal requirements, sites should not demand their users' attention to answer questions that aren't important enough to justify the cost to those users.

File: index.html [hunk 2]
```diff
@@ -549,6 +549,14 @@
...
+some action that is treated as <q>consent</q> without obtaining the person's actual, informed [=consent=].
```
Comment (npdoty) [inline index.html:553]: I don't know of any laws that require companies to request actions that do not demonstrate consent. That many companies have skirted some laws, or interpreted laws in ways that are harmful to users, is not the same as a legal requirement to harass users with meaningless dialogs.

**Comment by jyasskin** [comment]
We agreed to close this today.
